bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -25,7 +25,7 @@
 import numpy as np
 import quaternion
 import rospy
-import sophus as sp
+import sophuspy as sp
```

Comment:
this needs to be consolidate across machines

Reply:
I will wrap it in try-catch to make it work for both the cases